lib/amplitude_api/event.rb

```diff
@@ -50,6 +55,7 @@ def to_hash
       serialized_event[:event_properties] = event_properties
       serialized_event[:user_properties] = user_properties
       serialized_event[:time] = formatted_time if time
+      serialized_event[:ip] = ip if ip
```

Comment:
Won't `ip` always be at least an empty string here? Either this should check for that, or we should default to nil in `initialize`